homeassistant/components/aws/__init__.py

```diff
@@ -42,9 +43,9 @@
         DOMAIN: vol.Schema(
             {
                 vol.Optional(
-                    ATTR_CREDENTIALS, default=DEFAULT_CREDENTIAL
+                    CONF_CREDENTIALS, default=DEFAULT_CREDENTIAL
                 ): vol.All(cv.ensure_list, [AWS_CREDENTIAL_SCHEMA]),
-                vol.Optional(CONF_NOTIFY): vol.All(
+                vol.Optional(CONF_NOTIFY, default=[]): vol.All(
```

Comment:
that share the same list for all configs. Use `list()`

Reply:
I don't understand it. 

CONF_NOTIFY can refer to one of CONF_CREDENTIALS, or none. These two configurations don't have necessary relationship. Next step I will move polly tts and route53 to here, their relationship will be more dynamic.